oak/common/BUILD

```diff
@@ -111,6 +111,8 @@ cc_binary(
         "//oak/common:utils",
```

Comment:
Is app_config still needed as a dependency?

Reply:
Yes, it uses `oak::utils::read_file`

Reply:
Sorry, GitHub only let me comment on the wrong line: I meant `//oak/common:app_config` on the line above.

Reply:
It's needed, since the serializer uses `oak::WriteConfigToFile`:
https://github.com/project-oak/oak/blob/e6e6b5532017f11d3f624a279e4e2e80ca14959b/oak/common/app_config.h#L33-L34